tng-sec-gtw/default-ssl.conf

```diff
@@ -38,4 +38,24 @@ server {
                 proxy_pass_request_body on;
                 proxy_hide_header Public-key-pins;
         }
+	
+	location /api/v1/projects {
+                resolver 127.0.0.11;
+                set $api http://tng-sdk-portal-backend{DOMAIN}:5098$uri$is_args$query_string;
+                proxy_pass $api;
```

Comment:
Same here: change the variable $api by $projects to give a unique variable for this url